### PR TITLE
LRQA-71055 | master

### PR DESCRIPTION
**Pull request by alessandromalves**
https://issues.liferay.com/browse/LRQA-71055

### CODE REVIEW

**Comment by alessandromalves** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImport.testcase [hunk 1]
```diff
@@ -269,6 +269,7 @@ definition {
 			value1 = "Unable to validate referenced document because it cannot be found with the following parameters");
 	}
 
+	@description = "Verify if is possible edit an exported template."
```
Comment (weller0246) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImport.testcase:272]: typo error, please fix it

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImport.testcase [hunk 2]
```diff
@@ -516,6 +517,7 @@ definition {
 		AssertConsoleTextNotPresent(value1 = "NoSuchFileEntryException");
 	}
 
+	@description = "Verify if is possible export and import a DDL via Global."
```
Comment (weller0246) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImport.testcase:520]: Typo error, please fix it

**Review by weller0246 — CHANGES_REQUESTED** [review]
Please look through the descriptions, and fix the grammar issues (like if is possible, verify is possible, etc). 
Another point is that I don't think we need to delete the "use case" wording from the description as we use it in every test case. ( cc @gaborlovasdotcom  please add your opinion about this).
And one last thing: Please use periods to separate when describing which ticket the current test case is connecting to and then the description, like this: "This is a use case for LPS-32577. Add user inherited roles as SAML attribute statements."

**Comment by alessandromalves** [comment]
ci:test:sf

**Review by gaborlovasdotcom — APPROVED** [review]
LGTM

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Comment by gaborlovasdotcom** [comment]
fyi: @JoyceWang08211

**Pull request closed without merging**